### PR TITLE
UIDEXP-33: Change translation title to fix typo

### PR DESCRIPTION
**Pull request by Yuliia-Dovhal**
## Purpose:
Redeploy didn't change the string of translation, so I had to change the title

### CODE REVIEW

**Comment by YevheniiMaltsev** [comment]
Guys, do we really need to do so many changes just to apply 2 translations?

**Comment by Yuliia-Dovhal** [comment]
@YevheniiMaltsev 
I guess, yes.
Because without changing translation title, redeploy didn't catch and apply changes
Renaming only one title will look weird, so I renamed both for marc and json options
Unit tests, on my opinion, should be clearable in understanding what exactly is covered. So I renamed there too.
I think this is not a big problem, that I renamed in many places

File: translations/ui-inventory/en.json [hunk 1]
```diff
@@ -436,8 +436,8 @@
   "item.status.orderClosed": "Order closed",
   "inTransitReport": "In transit items report (CSV)",
   "saveInstancesUIIDS": "Save instances UUIDs",
-  "exportInstancesMARC": "Export instances (MARC)",
-  "exportInstancesJSON": "Export instances (JSON)",
+  "exportInstancesMrc": "Export instances (MARC)",
+  "exportInstancesJson": "Export instances (JSON)",
```
Comment (VictorSoroka1) [inline translations/ui-inventory/en.json]: Let's change it to be `exportInstancesInMARC` and `exportInstancesInJSON`

File: test/bigtest/interactors/inventory.js [hunk 2]
```diff
@@ -22,12 +22,12 @@ import {
   isSaveInstancesUIIDsBtnDisabled = property('#dropdown-clickable-get-items-uiids', 'disabled');
   isSaveInstancesUIIDsIconPresent = isPresent('#dropdown-clickable-get-items-uiids [class*=icon-save]');
   isTransitItemsReportIconPresent = isPresent('#dropdown-clickable-get-report [class*=icon-report]');
-  exportInstancesMARCBtnIsVisible = isVisible('#dropdown-clickable-export-marc');
-  isExportInstancesMARCBtnDisabled = property('#dropdown-clickable-export-marc', 'disabled');
-  isExportInstancesMARCIconPresent = isPresent('#dropdown-clickable-export-marc [class*=icon-download]');
-  exportInstancesJSONBtnIsVisible = isVisible('#dropdown-clickable-export-json');
-  isExportInstancesJSONBtnDisabled = property('#dropdown-clickable-export-json', 'disabled');
-  isExportInstancesJSONIconPresent = isPresent('#dropdown-clickable-export-json [class*=icon-download]');
+  exportInstancesMrcBtnIsVisible = isVisible('#dropdown-clickable-export-marc');
```
Comment (VictorSoroka1) [inline test/bigtest/interactors/inventory.js]: I think in this file changes are not needed.

File: translations/ui-inventory/en.json (hunk 1 above)
Comment (Yuliia-Dovhal) [reply translations/ui-inventory/en.json]: ok

File: test/bigtest/interactors/inventory.js (hunk 2 above)
Comment (Yuliia-Dovhal) [reply test/bigtest/interactors/inventory.js]: ok

**Pull request merged**